### PR TITLE
Remove Fomantic dimmer module

### PR DESCRIPTION
**Pull request by silverwind**
I think the only place where these could have been used were the basic modals which were removed, so let's remove this module.

### CODE REVIEW

File: web_src/fomantic/semantic.json [hunk 1]
```diff
@@ -25,7 +25,6 @@
     "button",
     "checkbox",
     "container",
-    "dimmer",
```
Comment (delvh) [inline web_src/fomantic/semantic.json:28]: A quick search showed that we can also remove line `55` in `web_src/fomantic/theme.config.less` (at least that's how I understand this file)

**Review by delvh — APPROVED** [review]

File: web_src/fomantic/semantic.json (hunk 1 above)
Comment (delvh) [reply web_src/fomantic/semantic.json:28]: By the way, why do we still have less?

Comment (silverwind) [reply web_src/fomantic/semantic.json:28]: Fomantic's source is Less and we compile that to CSS and vendor the output files. Theme config has no influence on compiled CSS, but I guess we can remove it there as well.

Comment (silverwind) [reply web_src/fomantic/semantic.json:28]: So yes, it's definitely pointless to modify theme config. The file contains a bunch of modules we do not even build, but it really does not matter. We only consume `semantic.css` and `semantic.js` output files.

**Comment by silverwind** [comment]
Modal still depends on it, need to check how to workaround/fix.

**Comment by silverwind** [comment]
I'm afraid it's not an easy removal as the modal moudle has a tight integration with it.

**Pull request closed without merging**